### PR TITLE
#CP-35 feat: update profile

### PR DESCRIPTION
**Pull request by Abdulkeza**
### What does this PR do?
- Update a user profile by using the backend endpoints

### Description of Task to be completed

- [x] The operator and driver should be able to update information about their own profile.
- [x] All Requests should be sent to the backend and return appropriate responses
- [x] Appropriate error messages are displayed on unsuccessful attempts.

### How should this be manually tested?
```
git clone https://github.com/atlp-rwanda/phantom-be-codebandits
cd phantom-be-codebandits
git fetch origin
git checkout ft-update-profile-cp-35
npm install
//configure the .env according to the .env.sample file
// start the database
npm run dev
// open browser or postman (Check linked public workplace for testing)
go to profile and update
```
### Any background context you want to provide?
- N/A

### What are the relevant pivotal tracker stories?
[DJacC8rQ](https://trello.com/c/DJacC8rQ/35-update-profile)

### CODE REVIEW

File: src/pages/profiles/OperatorProfile.js [hunk 1]
```diff
@@ -1,19 +1,52 @@
...
+      setloading(true);
+      try {
+        const response = await axiosBase.get('/accounts/profile');
+
+        setUserInfo(response.data.data.user);
+        setPublicInfo(response.data.data);
+        setloading(false);
+      } catch (error) {
+        toast(error.message, { type: 'error' });
+      } finally {
```
Comment (Yvad60) [inline src/pages/profiles/OperatorProfile.js]: add a toast for errors provided by the backend when the request has a fail status

File: src/utils/Api.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ import getLanguage from './getLanguage.js';
 
 const host = `${window.location.protocol}//${window.location.host}`;
 
-const base = 'https://phantom-be-codebandits-staging.herokuapp.com/api/v1';
+const base = 'https://phantom-codebandits.herokuapp.com/api/v1';
 const axiosBase = axios.create({
   baseURL: base,
```
Comment (Yvad60) [inline src/utils/Api.js]: remember to change the URL back to its initial value after the PR on the backend Repo is merged

**Review by Yvad60 — DISMISSED** [review]
nice work Mr. Andul. for the integration of profile updates and adding image support 
take a look at the feedback below

**Review by feyton — DISMISSED** [review]
### Approval
This feature is a great addition to the application and is well implemented. As this feature matches the on the backend. This is great work

**Review by Yvad60 — DISMISSED** [review]
Nice work @Abdulkeza now it's ready for merge

**Review by samyshyaka — DISMISSED** [review]
Great implementation @Abdulkeza, I believe the PR is ready to be merged

**Review by feyton — APPROVED** [review]

**Review by Yvad60 — APPROVED** [review]
Approved!

**Pull request merged**